docs/contributing.rst

```diff
@@ -8,46 +8,114 @@ For a more practical guide to the tedana development, please see our
-Every member of the ``tedana`` community is encouraged to comment on these processes and suggest improvements.
-
-As the first interim `Benevolent Dictator for Life (BDFL)`_,
-Elizabeth DuPre is ultimately responsible for any major decisions pertaining to ``tedana`` development.
-However, all potential changes are explicitly and openly discussed in the described channels of
-communication, and we strive for consensus amongst all community members.
-
-.. _Benevolent Dictator for Life (BDFL): https://en.wikipedia.org/wiki/Benevolent_dictator_for_life
-
 Code of conduct
 ```````````````
 
-All ``tedana`` community members are expected to follow our `code of conduct`_
-during any interaction with the project.
+All ``tedana`` community members are expected to follow our code of conduct
+during any interaction with the project. `The full code of conduct is here`_.
 That includes---but is not limited to---online conversations,
 in-person workshops or development sprints, and when giving talks about the software.
 
 As stated in the code, severe or repeated violations by community members may result in exclusion
 from collective decision-making and rejection of future contributions to the ``tedana`` project.
 
-.. _code of conduct: https://github.com/ME-ICA/tedana/blob/master/CODE_OF_CONDUCT.md
+.. _The full code of conduct is here: https://github.com/ME-ICA/tedana/blob/master/CODE_OF_CONDUCT.md
+
+Scope of tedana
+```````````````
+tedana is a collection of tools, software and a community related to echo time
+(TE) dependent analyses. The umbrella of tedana covers a number of overlapping,
+but somewhat distinct ideas related to multi-echo analysis. This scope covers
+collecting multi-echo data (Acquisition), combining those echoes together
+(Combining), with optional noise removal (Denoising), inspecting the outputs
+(Visualization) and answering multi-echo related questions (Community). In
+general, preprocessed data is pushed through tedana, producing outputs that
+are ready for further analyses. 
+
+Acquisition
+-----------
+
+While the development of multi-echo sequences is far beyond the current scope
+of tedana, the tedana community is committed to providing guidelines on current
+multi-echo implementations. This will include both specific instructions for
+how to collect multi-echo data for multiple vendors as well as details about
+what types of data have been collected thus far. These details are subject to
+change, and are intended to provide users with an idea of what is possible,
+rather than definitive recommendations.
+
+Our focus is on functional MRI, including both magnitude and phase data,
+however we understand that quantitative mapping has the potential to aid in
+data processing. Thus, we believe that some details on non-functional MRI
+acquisitions, such as detailed T2* mapping, may fall within the scope of
+tedana. Acquisition related details can be found in the tedana Documentation.
```

Comment:
We should cross-ref to the specific section, here.

Reply:
RTD question. https://tedana.readthedocs.io/en/latest/acquisition.html the best link or is there a way to make an internal link that would self-reference the correct location in the documentation?

Reply:
I guess that link's fine given that it's a reference to "Acquisition related details".